pkgs/applications/misc/logseq/default.nix

```diff
@@ -8,7 +8,10 @@
 , nix-update-script
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: let
+  inherit (finalAttrs) pname version src appimageContents;
```

Comment:
I'd rather replace ${pname} with ${finalAttrs.pname} below than inherit here.

Reply:
there are indeed many occurrences of these variables. i don't see the benefit of adding `finalAttrs.` everywhere. what is the benefit of adding that boilerplate?

Reply:
> there are indeed many occurrences of these variables.

Ah, indeed.